### PR TITLE
Added EA

### PR DESCRIPTION
**Pull request by iMonZ**
Added EA to the list

### CODE REVIEW

File: quirks/password-rules.json [hunk 1]
```diff
@@ -200,5 +200,8 @@
     },
     "xfinity.com": {
         "password-rules": "minlength: 8; maxlength: 16; required: lower, upper; required: digit;"
+    },
+    "signin.ea.com": {
+     "password-rules": "minlength: 8; maxlength: 64; required: lower, upper; required: digit;"
```
Comment (rmondello) [inline quirks/password-rules.json:205]: Can you make the indentation on this line match the other similar lines in the file? After that, I’m happy to merge. :)

File: quirks/password-rules.json [hunk 2]
```diff
@@ -202,6 +202,6 @@
         "password-rules": "minlength: 8; maxlength: 16; required: lower, upper; required: digit;"
     },
     "signin.ea.com": {
-     "password-rules": "minlength: 8; maxlength: 64; required: lower, upper; required: digit;" 
+        "password-rules": "minlength: 8; maxlength: 64; required: lower, upper; required: digit;"
```
Comment (rmondello) [inline quirks/password-rules.json:205]: Thanks for fixing this! It looks like EA allows a few special characters that aren’t required, like `!@#-^&*=+;:`.

Can you augment these rules with something like `allowed: [!@#-^&*=+;:]`, and re-validate the rule is valid using the developer tool? Then I think we’ll be good to go here!

**Comment by rmondello** [comment]
That’s right! One last thing.

You have:
`minlength: 8; maxlength: 64; required: lower, upper; required: digit;, allowed: [!@#-^&*=+;:];`

I think we want:
`minlength: 8; maxlength: 64; required: lower, upper; required: digit; allowed: [!@#-^&*=+;:];`

(See stray comma.)

File: quirks/change-password-URLs.json [hunk 3]
```diff
@@ -1,4 +1,5 @@
 {
     "tripit.com": "https://tripit.com/account/edit/section/change_password",
     "twitch.tv": "https://www.twitch.tv/settings/security"
+    "ea.com": "https://myaccount.ea.com/cp-ui/security/index"
```
Comment (rmondello) [inline quirks/change-password-URLs.json:4]: Previous line needs a comma.

**Comment by rmondello** [comment]
No worries! Merged! Thank you!

**Comment by iMonZ** [comment]
Tomorrow I could add a few other websites especially that where the password manager is not working! Have a great week and stay healthy

**Review by ame1234 — COMMENTED** [review]
Authoring is not correct no merging allowed

**Pull request merged**